granulate_utils/metadata/databricks_client.py

```diff
@@ -11,21 +11,23 @@
 
 import requests
 
+from granulate_utils.exceptions import DatabricksJobNameDiscoverException
+from granulate_utils.metadata import Metadata
+
 HOST_KEY_NAME = "*.sink.ganglia.host"
-DATABRICKS_DEPLOY_CONF_PATH = "/databricks/common/conf/deploy.conf"
 DATABRICKS_METRICS_PROP_PATH = "/databricks/spark/conf/metrics.properties"
 CLUSTER_TAGS_KEY = "spark.databricks.clusterUsageTags.clusterAllTags"
-JOB_NAME_KEY = "RunName"
 SPARKUI_APPS_URL = "http://{}/api/v1/applications"
 REQUEST_TIMEOUT = 5
 DEFAULT_WEBUI_PORT = 40001
-MAX_RETRIES = 90
+MAX_RETRIES = 300
+RETRY_INTERVAL = 1
```

Comment:
```suggestion
RETRY_INTERVAL_S = 1
```
Makes it clear "1 what"? second? ms?

Reply:
039bc37